drivers/test.go

```diff
@@ -74,11 +74,14 @@ func init() {
 		State:   init_spec.ConstructState(EmptyMapCid, "chain-validation"),
 	}
 
+	firstRewardState := reward_spec.ConstructState()
+	firstRewardState.LastPerEpochReward = big_spec.NewInt(100000000000000000)
```

Comment:
this is temporary, correct? Can you file an issue to track this if so?

Reply:
Yup, I'll open a specs-actors issue for it (and one here to link to that, I guess?)

Reply:
perfect